### PR TITLE
BraveRoundTimeStamps RELEASE 100%

### PR DESCRIPTION
**Pull request by atuchin-m**
Resolves https://github.com/brave/brave-variations/issues/1214

### CODE REVIEW

File: seed/seed.json
Comment (atuchin-m) [inline seed/seed.json:2946]: The feature was disabled here: https://github.com/brave/brave-core/pull/25877#issuecomment-2402130204

**Review by kjozwiak — APPROVED** [review]
Uplift into `main/production` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
